tools/bootnode-query/main.go

```diff
@@ -1,94 +0,0 @@
-// Bootstrap / DHT query tool
```

Comment:
we should keep this tool, and retool it for discoveryV5. Performing a walk through the dht is still useful